LICENSE

```diff
@@ -217,3 +200,149 @@ solely pursuant to the terms of the relevant commercial agreement.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+================================================================================
+
+For the `es/upstream` directory:
```

Comment:
I'm not sure this sentence is necessary because es/upstream is not part of this repository. It's pulled in via Git during the build process. We also don't mention how the other dependencies go into Crate (e.g. via Gradle).

Reply:
this is an edge case for sure, but I think we should document git submodules as if they were bundled by the source

we can add a note saying that it's a submodule

the stuff re gradle will be handled in the finished changeset! ty!

Reply:
still some work to do on this. just on the backburner for a few days